client/src/components/pages/application/Steps/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useHistory } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import Limitation from '../../../common/limitation';
+import Typography from '../../../common/Typography';
+import Button from '../../../common/Button';
+import backGround from '../../../../assets/images/backGround.svg';
+
+import './style.css';
+
+const getSteps = async () => {
+  const { data } = (await axios.get('/api/v1//steps')).data;
```

Comment:
there is duplicate slashes here, please omit it, also fix steps route in the App--> index file its `steps` component not `Login`. 
```suggestion
  const { data } = (await axios.get('/api/v1/steps')).data;
```